frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -201,6 +255,19 @@ const Details_ = ({ flags, obj: pvc }) => {
               )}
             </dl>
           </div>
+          <div className="hidden-xs hidden-sm col-md-4 col-xl-2 co-pvc-donut">
+            {totalCapacityMetric && !loading && (
+              <ChartDonut
```

Comment:
In my above screen shot, you can see that the donut 'thickness' and font size are increasing as the chart becomes larger. This is not a desired behavior. The font size and donut thickness should always be constant, while the radius grows to fill the space. But we also want to keep the total size within a reasonable and consistent range (see my previous comment about the max-width and height we are setting through css).

Reply:
So I tested it on my local system and the CSS that wraps the parent component is not letting it grow at all. The `.co-pvc-donut` class.

Reply:
Even the chart is a static size, you will need to set the width and height to match the parent. This is so that the svg coordinate space matches the html coordinate space (1px in svg = 1px in html). The ChartDonut component has a default width/height of 130px. If you don't change that, those 130px just get "stretched out" to fill the available space in the parent html element. You'll notice that the text in your donut is much larger than the rest of the standard text on the page. That's not because the font size is larger, it's because the svgs coordinate space is not aligned to the html. Since the html parent size 325px and the svg size is 130px, 1px in your svg space scales to 2.5px in the html space.

Reply:
Thank you for the explanation syncing the bounding div and the svg size.